Userland/Libraries/LibSQL/AST/Insert.cpp

```diff
@@ -11,6 +12,19 @@
 
 namespace SQL::AST {
 
+static bool does_value_data_type_match(const SQLType expected, const SQLType actual)
```

Comment:
Is there any need for these types to be const here? Since you are taking them by value it does not really matter so just remove `const`. Alternatively you can take by const reference if the objects are large.

**IF** you keep them:
Please use east const see https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md.

```suggestion
static bool does_value_data_type_match(SQLType const expected, SQLType const actual)
```